packages/react-testing/src/compat.ts

```diff
@@ -0,0 +1,11 @@
+import {ReactInstance, Fiber} from './types';
+
+export function getInternalFiber(instance: ReactInstance): Fiber {
```

Comment:
Silly nitpick - perhaps run with the most recent name `getInternals` so our terminology matches the most recent react naming